### PR TITLE
Features/cell design

### PR DESCRIPTION
**Pull request by gilemos**
hey guys! More design changes coming \o/
Here is how it looks like now, see if you like it:

Screen Shot 2019-08-01 at 1 49 28 PM
Screen Shot 2019-08-01 at 1 49 35 PM

### CODE REVIEW

**Comment by gilemos** [comment]
Things that I still want to do:
 - put a big image of the city on home table view
- increase the borders of the collection view in more options view
- change the style of the buttons in navigation controller

File: Travel Scheduler/View Controllers/HomeCollectionViewController.m [hunk 1]
```diff
@@ -220,17 +214,23 @@ - (NSIndexPath *)tableView:(UITableView *)tableView willSelectRowAtIndexPath:(NS
...
     } else if (cellNum == 2) {
+        moreOptionViewController.stringType = @"Restaurants";
+        moreOptionViewController.correctType = @"restaurant";
+        moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[@"lodging"];
+    } else if (cellNum == 3) {
         moreOptionViewController.stringType = @"Hotels";
+        moreOptionViewController.correctType = @"lodging";
         moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[@"lodging"];
     }
+    moreOptionViewController.places = self.hub.dictionaryOfArrayOfPlaces[moreOptionViewController.correctType];
```
Comment (aliu917) [inline Travel Scheduler/View Controllers/HomeCollectionViewController.m:229]: You seem to be setting moreOptionViewController.places individually within all the if statements (and again here). I think you can remove all the lines inside the if statements to be less repetitive.

**Review by aliu917 — APPROVED** [review]
Looks good! Nice design changes!

File: Travel Scheduler/View Controllers/HomeCollectionViewController.m (hunk 1 above)
Comment (gilemos) [reply Travel Scheduler/View Controllers/HomeCollectionViewController.m:229]: omg I was planning on erasing that and I forgot! Thanks for noticing that!

**Pull request merged**